generators/entity/index.js

```diff
@@ -78,6 +85,7 @@ module.exports = yeoman.generators.Base.extend({
       this.props = props;
       // To access props later use this.props.someOption;
       this.enableAudit = props.enableAudit;
+      this.auditFramework = this.config.get('auditFramework');
```

Comment:
Do we need to fetch this each time, cant we get it ince at the beginning and check using the variable after that